app/models/mixins/file_depot_mixin.rb

```diff
@@ -24,6 +24,21 @@ def verify_depot_settings(settings)
       res
     end
 
+    def verify_depot_settings_queue(userid, options)
+      task_opts = {
+        :action => "Verify #{display_name} Credentials",
```

Comment:
I don't think this is i18n compliant.  @mzazrivec can you remind me of the right way to do this?

Reply:
I _think_ it's 

```suggestion
        :action => raise _("Verify %{display_name} Credentials") % {:display_name => display_name}
```

Reply:
@Fryguy correct

Reply:
This `action` would be logged into the log file and saved into task's `message` column. Do we really need to make it I18N compliant? @Fryguy

Reply:
If the message ends up in a log file, then no, we're not translating that.